README.markdown

```diff
@@ -152,7 +152,7 @@ Upstream bar.com
         127.0.0.1:12356 up
 ```
 
-If an upstream has no health checkers, then it will be marked by `(NO checkers)`, as in
+If you turn on lua_code_cache off or an upstream has no health checkers, then it will be marked by `(NO checkers)`, as in
```

Comment:
I don't think we should document the behavior of `lua_code_cache off`. It was for dirty and quick development for *individual* nginx requests only. anything that need to span multiple nginx requests (like timers, cosocket connection pools, semaphores) won't work anyway in `lua_code_cache off`.